NuGet/ChakraCore.Debugger.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>ChakraCore.Debugger</id>
+    <version>$version$</version>
+    <description>Debugging companion library for the ChakraCore JavaScript engine.</description>
+    <authors>Microsoft</authors>
+    <projectUrl>https://github.com/Microsoft/ChakraCore-Debugger</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
```

Comment:
We should include appropriate license information here; probably something like

`<licenseUrl>https://github.com/Microsoft/ChakraCore-Debugger/blob/master/LICENSE</licenseUrl>`

Reply:
Also a copyright statement, something like `<copyright>Copyright (C) 2018 Microsoft</copyright>`

Reply:
Done.